falco.yaml

```diff
@@ -25,50 +25,64 @@
 #
 #     (Falco command-line arguments)
 #     (Falco environment variables)
-# Falco config files
-#     configs_files
+# Falco config files settings
+#     config_files [Stable]
+#     watch_config_files [Stable]
 # Falco rules files
-#     rules_files
+#     rules_files [Stable]
+# Falco rules
+#     rules [Incubating]
 # Falco engine
-#     engine
+#     engine [Stable]
 # Falco plugins
-#     load_plugins
-#     plugins
-# Falco config files
-#     watch_config_files
+#     load_plugins [Stable]
+#     plugins [Stable]
 # Falco outputs settings
```

Comment:
Note for eg: 1.0.0: we might add an `outputs.` namespace for these settings.
Similarly, for all the others, eg:
* `services.{grpc,webserver}`
* log.{stderr,syslog,level,libs}`
...